test/Performance/README.md

```diff
@@ -1,21 +1,20 @@
 ## Setup Environment on a clean machine
 

```

Comment:
it would be nice to add links telling users where to get the PHP zip and also our drivers